src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx

```diff
@@ -33,21 +36,23 @@ function QuickbooksOutOfPocketExpenseConfigurationPage({policy}: WithPolicyProps
                     <MenuItemWithTopDescription
                         title={exportEntity ? translate(`workspace.qbo.${exportEntity}`) : undefined}
                         description={translate('workspace.qbo.exportAs')}
+                        error={exportEntity && (showTaxError || showLocationError) ? translate(`workspace.qbo.${exportEntity}Error`) : undefined}
                         onPress={() => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_OUT_OF_POCKET_EXPENSES_SELECT.getRoute(policyID))}
-                        brickRoadIndicator={undefined}
+                        brickRoadIndicator={brickEntityRoadIndicator ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
```

Comment:
This looks incorrect - the error message should be sitting below the red dot and the arrow. Can you adjust this and report back?

Reply:
@shawnborton hmm - seems like generic component is not expect to fit correct space (not expect to be long):
<img width="372" alt="Screenshot 2024-04-18 at 13 19 49" src="https://github.com/Expensify/App/assets/28352309/bdc7f295-bb06-4ad9-b196-99f294d8e6b6">

Refactoring this one - may cause potential regression - do we want to have that refactoring as a part of current PR? or make it as a follow up UI clean up?

Reply:
Hmm how much refactoring are we talking about? I feel like we should do it the correct way the first time instead of implementing something that isn't correct.

Reply:
yeah - but these component for MenuItem - is used everywhere - probably will need to check all places where we have that error field. No ideas why it was build in such way.
Problem that this red line has absolute position - and trying to fit it based on text lines will may cause UI issues
`        menuItemError: {
            position: 'absolute',
            bottom: -4,
            left: 20,
            right: 20,
        },`

Reply:
Hmm yeah, we should just be using flex styles to allow the error message to grow and shrink as needed. 

cc @cead22 - does this sound familiar from the violations project?

Reply:
Yeah it looks familiar. For violations we first added the violation messages on a separate element, and then later changed that and put the violation messages in the existing element that we put errors on the `error` prop of `MenuItemWithTopDescription` which is where we were already putting error messages for those items

Reply:
Yeah but error prop is not rendering below the red dot and the arrow. Seems we already need a follow for this one. Probably for now we can use same way - just using error prop.